content/posts/June-2020.md

```diff
@@ -10,10 +10,23 @@ Hello everyone, during this week we didn't manage to implement any significant n
 
 ### packit
 
-* We have fixed a formatting issue when packit prints installation instructions for using builds from a copr repo (by [@lachmanfrantisek](https://github.com/lachmanfrantisek), [#852](https://github.com/packit-service/packit/pulls/852).
-* You are able to set up builds when you push to a branch, we have set this up for master branch of packit, feel free to get inspired (by [@lachmanfrantisek](https://github.com/lachmanfrantisek), [#851](https://github.com/packit-service/packit/pulls/851).
+* We have fixed a formatting issue when packit prints installation instructions for using builds from a copr repo (by [@lachmanfrantisek](https://github.com/lachmanfrantisek), [#852](https://github.com/packit-service/packit/pulls/852)).
+* You are able to set up builds when you push to a branch, we have set this up for master branch of packit, feel free to get inspired (by [@lachmanfrantisek](https://github.com/lachmanfrantisek), [#851](https://github.com/packit-service/packit/pulls/851)).
 
 
 ### packit-as-a-service
 
 * We have run into multiple problems when you set up jobs for PRs and branch pushes - there were multiple fixes for this, packit-service wasn't able to work well with the fact that there are multiple jobs defined for the same job type - e.g. build.
+
+
+## Week 24 (June 8th - June 11th)
+
+### packit
+
+* [Tomas](https://github.com/TomasTomecek) re-implemented overriding of package config per job, which would also enable you to set up different configurations for builds, releases, etc. ([#858](https://github.com/packit-service/packit/pull/858), [#859](https://github.com/packit-service/packit/pull/859)).
```

Comment:
If I understand it correctly, this is not ready to be used yet, as it requires packit-service/packit-service#676. So maybe rephrase this as "... config per job. This will enable overriding configuration for builds and releases, for example, once the corresponding changes are merged in Packit-as-a-Service."

Reply:
amended